bundles/org.openhab.binding.mqtt.espmilighthub/README.md

```diff
@@ -1,4 +1,4 @@
-# EspMilightHub Binding
+# esp8266_milight_hub Binding
```

Comment:
Don't have a better suggestion, but i wonder if this is the proper way to name the binding.